packages/gallery/package.json

```diff
@@ -37,13 +37,13 @@
   },
   "dependencies": {
     "@vimeo/player": "2.8.2",
-    "hls.js": "^1.1.5",
+    "hls.js": "1.4.12",
     "pro-gallery-blueprints": "5.0.72",
     "pro-gallery-lib": "5.0.72",
     "pro-layouts": "5.0.72",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-player": "^2.6.2",
+    "react-player": "2.11.0",
```

Comment:
isn't the change in 'MediaItem.tsx' solves the issue? why do we need to fix those packages?

Reply:
@shmuelhizmi why without '^'

Reply:
@shmuelhizmi why without '^'?